hera_qm/tests/test_ant_metrics.py

```diff
@@ -37,26 +37,50 @@ def test_mean_Vij_metrics(self):
                                                 self.ants, self.bls, rawMetric=True)
         self.assertEqual(mean_Vij, {(1, 'y'): 1.0, (1, 'x'): 1.0, (0, 'x'): 1.0,
                                     (0, 'y'): 1.0, (2, 'x'): 1.0, (2, 'y'): 1.0})
+        args = [self.data, self.pols, self.antpols, self.ants, self.bls]
+        zs = uvtest.checkWarnings(ant_metrics.mean_Vij_metrics, args, nwarnings=6,
```

Comment:
Oh I see. In that case, I think we should probably change it. My biggest concern is that these warnings might be hiding some more sinister bugs, so if we could eliminate those warnings that would be great. Maybe we can have simple-but-not-uniform data, something like an even distribution of 1s, 2s, and 3s? So our median would be 2, and the modified z-score would be calculable. (Haven't thought through all the specifics, but I can cook up some simple data with a known result if you'd like me to.)